includes/functions.php

```diff
@@ -1270,7 +1270,7 @@ function wpum_get_plugins() {
 		$dirname = strtolower( dirname( $plugin_path ) );
 		if ( strstr( $dirname, 'wpum-' ) && strstr( $plugin_data['AuthorURI'], 'wpusermanager.com' ) ) {
 			$plugins[ $plugin_path ]['Type'] = 'add-on';
-			$license_active                  = __wpum_get_active_license_info( WPUM_License::get_short_name( $plugin_data['Name'] ) );
+			$license_active                  = false; // FIXME __wpum_get_active_license_info( WPUM_License::get_short_name( $plugin_data['Name'] ) );
```

Comment:
`__wpum_get_active_license_info()` and `WPUM_License::get_short_name()` do not exist.